sunpy/map/mapbase.py

```diff
@@ -773,8 +777,17 @@ def _validate_meta(self):
             CUNIT1, CUNIT2, WAVEUNIT
 
         """
+        msg = ('Image coordinate units for axis {} not present in metadata. '
```

Comment:
Note to self: this message needs changing to tell users how to load data if the metadata is missing.